docs/man/minfo.md

```diff
@@ -49,7 +49,7 @@ OPTIONS
   Authenticate as user under account.
```

Comment:
Can't cite it above, but shouldn't you add the "-j, --json" option to this man page?

Reply:
Yes! Thanks for catching that.